order-form-ui/src/OrderForm.js

```diff
@@ -211,23 +199,15 @@ class OrderForm extends React.Component {
 					</div>
 					<div className="form-group" id="tele">
 						<label htmlFor="tel_inpt">Telefonnummer<span>*</span>:</label>
-						<input type="tel" name="telNr" id="tel_inpt" onChange={this.handle_detail_update} className={'form-control ' + (this.is_telNr_valid ? '' : 'invalid')} aria-describedby="emailHelp" placeholder="070......."/>
+						<input type="tel" name="telNr" id="tel_inpt" onChange={this.handle_detail_update} className={'form-control ' + (this.is_telNr_valid ? '' : 'invalid')} aria-describedby="emailHelp" placeholder="07........"/>
 					</div>
 					<div className="form-group">
 						<label htmlFor="name_inpt">Namn<span>*</span>:</label>
-						<input type="text" name="name" id="name_inpt" onChange={this.handle_detail_update} className="form-control" aria-describedby="emailHelp" placeholder="Glen Glensson"/>
-					</div>
-					<div className="form-group" >
-						<label htmlFor="addr_inpt">Leveransaddress<span>*</span>:</label>
-						<input type="text" name="address" id="addr_inpt" onChange={this.handle_detail_update} className="form-control" aria-describedby="emailHelp" placeholder="Pizzagatan 123"/>
-					</div>
-					<div className="form-group" >
-						<label htmlFor="post_nr_inpt">Postkod<span>*</span>:</label>
-						<input type="text" name="postalCode" id="post_nr_inpt" onChange={this.handle_detail_update} className={'form-control ' + (this.is_postalCode_valid ? '' : 'invalid')} aria-describedby="emailHelp" placeholder="123 45"/>
+						<input type="text" name="name" id="name_inpt" onChange={this.handle_detail_update} className="form-control" aria-describedby="emailHelp" placeholder="Namn Efternamn"/>
 					</div>
 					<div className="form-group" >
-						<label htmlFor="code_inpt">Portkod:</label>
-						<input type="text" name="doorCode" id="code_inpt" onChange={this.handle_detail_update} className="form-control" aria-describedby="emailHelp" placeholder="0001"/>
+						<label htmlFor="pickup_inpt">Uthämtningsställe<span>*</span>:</label>
+						<input type="text" name="location" id="pickup_inpt" onChange={this.handle_detail_update} className="form-control" aria-describedby="emailHelp" placeholder="Meno Male"/>
```

Comment:
Change this to a drop-down menu since it has to be 1 of 3 diffrent places: Vasastan, Kungsholmen, Östermalm. check this out: https://www.w3schools.com/tags/tag_select.asp